webui/settings.py

```diff
@@ -137,33 +137,6 @@
 
 STATIC_URL = '/static/'
 
-logging.config.dictConfig({
```

Comment:
Was this just moved or also altered?

Reply:
one line was altered 

```
-  'level': 'INFO',
+ 'level': loglevel,
```

Reply:
from now on we will be able to tweak log level from pcw.ini w/o need to chage anything in PCW :eyeglasses: